.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
```

Comment:
Will this show error to user or `auto-format` the code? Like flake8 mentioned will show all the changes that should be done. I think instead of formatting, we should inform contributor that there are some changes that can be done and to use black and isort.

Reply:
Yes, makes sense.  We can run black with `--check` to only check and tell the contributor which files need changes.